File: VB/WebSite/App_Code/DetailObject.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web

Public Class DetailObject
	Public Sub New(ByVal key As Integer, ByVal value As String)
		Key = key
		Value = value
	End Sub
	Private privateKey As Integer
	Public Property Key() As Integer
		Get
			Return privateKey
		End Get
		Set(ByVal value As Integer)
			privateKey = value
		End Set
	End Property
	Private privateValue As String
	Public Property Value() As String
		Get
			Return privateValue
		End Get
		Set(ByVal value As String)
			privateValue = value
		End Set
	End Property
End Class

File: CS/WebSite/App_Code/DetailObject.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

public class DetailObject
{
    public DetailObject(int key, string value)
    {
        Key = key;
        Value = value;
    }
    public int Key { get; set; }
    public string Value { get; set; }
}